README.md

```diff
@@ -8,25 +8,81 @@ This repository contains the core of the application and the web services.
+                <eva.mongo.read-preference></eva.mongo.read-preference>
+                <eva.mongo.collections.annotation-metadata></eva.mongo.collections.annotation-metadata>
+                <eva.mongo.collections.annotations></eva.mongo.collections.annotations>
+                <eva.mongo.collections.features></eva.mongo.collections.features>
+                <eva.mongo.collections.files></eva.mongo.collections.files>
+                <eva.mongo.collections.variants></eva.mongo.collections.variants>
+            </properties>
+        </profile>
+    </profiles>
+
+    <servers>
+        <server>
+            <id>maven.oracle.com</id>
+            <username>your_oracle_user</username>
+            <password>your_oracle_encrypted_password</password>
+            <configuration>
+              <basicAuthScope>
+                <host>ANY</host>
+                <port>ANY</port>
+                <realm>OAM 11g</realm>
+              </basicAuthScope>
+              <httpConfiguration>
+                <all>
+                  <params>
+                    <property>
+                      <name>http.protocol.allow-circular-redirects</name>
+                      <value>%b,true</value>
+                    </property>
+                  </params>
+                </all>
+              </httpConfiguration>
+            </configuration>
+      </server>
+   </servers>
+</settings>
+```
+
+After setting up your maven security and maven settings, you can install the whole project and run the tests using the above "production" profile with `mvn clean install -Pproduction`. This will generate the ".war" files that you can deploy in Tomcat or other Java container. You have to set up the JNDI credentials in Tomcat (conf/context.xml), matching the "eva.evapro.datasource" and "eva.evapro.jdbc.url" properties.
+
+If you don't need to build dgva-server, you can do `mvn clean install -pl '!dgva-server'` to build the ".war" files of all the other modules only.
 
-After it has been compiled, if you just want to build the WAR, run `mvn package -DskipTests` and you should obtain a file to deploy in Tomcat or other Java container.
 
 Testing
 -------
 
-The tests implemented so far are integration (not unit) tests, so a working WAR file needs to be created first. The Jetty plugin for Maven has been included to ease the testing process.
+The tests implemented are both unit and integration tests. You can run them with `mvn test` from the root folder.
+
+If you are compiling and want to skip the tests, you can do `mvn clean install -Pproduction -DskipTests`.
 
-1. Fill the datasource information in the file `eva-server/src/main/webapp/WEB-INF/jetty-env.xml`
-2. Build the WAR file as described in the section above
-3. Run `mvn jetty:run` from the eva-server subfolder
-4. Run `mvn test` from the root folder
+For manual testing, you can deploy the ".war" files and go to the "swagger-ui.html" page to get an overview of the endpoints and run them manually.
```

Comment:
We could show the explicit URL to swagger-ui.html in localhost to make the check a bit easier.